linode/networkreservedips/framework_models.go

```diff
@@ -0,0 +1,69 @@
+package networkreservedips
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/linode/linodego"
+
+	"github.com/linode/terraform-provider-linode/v2/linode/helper"
+)
+
+type ReservedIPModel struct {
+	ID         types.String `tfsdk:"id"`
+	Region     types.String `tfsdk:"region"`
+	Address    types.String `tfsdk:"address"`
+	Gateway    types.String `tfsdk:"gateway"`
+	SubnetMask types.String `tfsdk:"subnet_mask"`
+	Prefix     types.Int64  `tfsdk:"prefix"`
+	Type       types.String `tfsdk:"type"`
+	Public     types.Bool   `tfsdk:"public"`
+	RDNS       types.String `tfsdk:"rdns"`
+	LinodeID   types.Int64  `tfsdk:"linode_id"`
+	Reserved   types.Bool   `tfsdk:"reserved"`
+}
+
+func (m *ReservedIPModel) FlattenReservedIP(
+	ctx context.Context,
+	ip linodego.InstanceIP,
+	preserveKnown bool,
+) diag.Diagnostics {
+	var diags diag.Diagnostics
```

Comment:
Looks like no diags is raised from this function. You can just remove it.

Reply:
I will look into it. I will remove it if it is not being triggered.